spark/dl/src/main/scala/com/intel/analytics/bigdl/visualization/README.md

```diff
@@ -8,113 +8,39 @@
 >You can find the wheel repository at https://pypi.python.org/pypi/tensorboard
 
 
-Simply, with one pip command you can install it in Ubuntu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
+Simply, with one pip command you can install it in Ununtu. But it is very likely that you will run into problems while installing. If you meet any error, please check the instructions below.
 
-If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information. 
+If you install it with Mac OS, similar in ubuntu, it is easy to install with pip. Notice that the wheels for mac do not require CPython to support wide unicode. If you meet any error, read the instructions below carefully. You may find your own solutions from these information.
+
```

Comment:
Delete the above to paragraph; simply say something like:

```
Please follow the instructions below to install TensorBoard; it has been tested on both Ubuntu and Mac OS. Please refer to the "Know Issues" section for possible errors (please note that CPython is not require to support wide unicode on Mac OS)
```

Two questions though:
1) Have we tested it on Redhat or CentOS?
2) Why do we call out CPython is not required on mac? Does it hurt to have CPython installed on mac?